k8s/test-environment/deploy-test.yaml

```diff
@@ -0,0 +1,196 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: testing-secrets 
+  namespace: ${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}
+type: Opaque
+stringData: 
+  DEBUG: "0"
+  ALLOWED_HOSTS: "${PR_NUMBER}-${CIRCLE_PROJECT_REPONAME}-test.geek.zone"
+  STRIPE_SECRET_KEY: "${TEST_STRIPE_SECRET_KEY}"
+  STRIPE_PUBLIC_KEY: "${TEST_STRIPE_PUBLIC_KEY}"
+  SAND_PRICE_ID: "price_1ImwyKJh8KDe9GPFrw4mWFai"
+  DONATION_PRODUCT_ID: "prod_HnmOmrwV7zDBfw"
+  GMAIL_APP_PASSWORD: "$GMAIL_APP_PASSWORD"
+  DATABASE_HOST: "postgres"
+  DATABASE_PORT: "5432"
+  DATABASE_NAME: "geekzone"
+  DATABASE_USER: "${TEST_POSTGRES_USER}"      
+  DATABASE_PASSWORD: "${TEST_POSTGRES_PASSWORD}"
+  CELERY_BROKER_URL: "${CELERY_BROKER_URL}"
+  #CELERY_BACKEND: "amqp://h3_ZfqkTGiGmQIU5U6Y5YMsC-hKz6CyL:aWP_KIM-ZIZFNmixt8GBbnigKM5oWPFb@172.20.159.27"
```

Comment:
Do we need this comment? We should avoided commented code in master

Reply:
I'll delete this PR and create a new one